src/vm/docs/mod.rs

```diff
@@ -241,6 +241,25 @@ Note: This function is only available starting with Stacks 2.1.",
 "#,
 };
 
+const IS_STANDARD_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: None,
+    signature: "(is-standard principal)",
+    description:
+        "Tests whether `principal` matches the current network type. That is, the network is either
```

Comment:
`principal` is the name of a type, so we should probably use a different word here.

Also, the copy should be more specific about what this means, like `represents a principal that can spend tokens`.  The thing is, principal addresses with unspendable version bytes will be accepted as literals (i.e. you can `stx-transfer?` to an address with a non-standard version byte); it's just that any tokens they receive won't be spendable.

Reply:
what do you think of the current wording?